WordPress/src/main/java/org/wordpress/android/ui/accounts/SignInFragment.java

```diff
@@ -631,7 +632,12 @@ protected void signInError(int messageId, String clientResponse) {
         AnalyticsTracker.track(Stat.LOGIN_FAILED);
         FragmentTransaction ft = getFragmentManager().beginTransaction();
         SignInDialogFragment nuxAlert;
-        if (messageId == org.wordpress.android.R.string.account_two_step_auth_enabled) {
+        if (!NetworkUtils.isNetworkAvailable(getActivity())) {
```

Comment:
> The condition if (!NetworkUtils.isNetworkAvailable(getActivity())) is not true, and the app will show the classic "we cannot log you in" error message.

Not sure we should display a "no network" message in case of a Timeout (we don't know if that comes from the client or the server), displaying a "timeout" message is not user friendly, so I guess the generic message is enough (and it's probably a rare case).